docs/functions/invoking-functions.md

```diff
@@ -0,0 +1,10 @@
+# Invoking functions
+
+You can use the `func invoke` command to send a test request to invoke a
+function either locally or on your Knative cluster.
+
+This command can be used to test that a function is working and able to receive HTTP requests and CloudEvents correctly.
+
+If your function is running locally, `func invoke` sends a test request to the local instance.
```

Comment:
If the function is running remotely, `func invoke` sends a test request to the remote instance on the cluster, unless it is also running locally.